src/cli.js

```diff
@@ -465,6 +473,14 @@ function cli (inputArgs) {
             // User explicitly did not pass in searchpath
             args.searchpath = conf.get('searchpath');
         }
+        if (args.production === undefined) {
+            // User explicitly did not pass in noprod
+            args.production = conf.get('production');
+        }
+        if (args.noprod === undefined) {
```

Comment:
lets not offer `noprod` in config. Just `args.production`. We only need noprod like you have in line 454. Used to figure out of args.production is true or false.